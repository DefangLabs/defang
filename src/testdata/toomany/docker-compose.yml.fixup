{
  "service1": {
    "command": null,
    "deploy": {
      "replicas": 1,
      "resources": {},
      "placement": {}
    },
    "entrypoint": null,
    "image": "example",
    "networks": {
      "default": null
    }
  }
}