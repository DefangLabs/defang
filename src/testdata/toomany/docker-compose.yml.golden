name: toomany
services:
  service1:
    image: example
    networks:
      default: null
networks:
  default:
    name: toomany_default
